source/localizable/tutorial/installing-addons.md

```diff
@@ -112,4 +112,4 @@ export default DS.JSONAPIAdapter.extend({
 
 ```
 
-If you were running `ember serve` in another shell, restart the server to include Mirage in your build.
+If you were running `ember serve` in another shell, restart the server to include Mirage in your build. Note that at this point of the tutorial, the data will be still being fetched from the `app/routes/rentals.js` file. It will be on the [Using Ember Data](https://guides.emberjs.com/v2.12.0/tutorial/ember-data/) section where a data model will be included and the data will be fetched from our new `mirage/config.js` file.
```

Comment:
Thanks for your comments. Totally agree in points 2 and 3. I  don't understand point 1, though. Why "new line after each sentence"? I can find many examples where several sentences are just separated by a ". "

So what do you exactly mean?